src/jquery.waitforimages.js

```diff
@@ -108,7 +108,8 @@
 
                     // If an `img` element, add it. But keep iterating in
                     // case it has a background image too.
-                    if (element.is('img:has-src')) {
+                    if (element.is('img:has-src') &&
+                        !element[0].hasAttribute('srcset')) {
```

Comment:
@alexanderdickson I don’t understand your advice. `[srcset]` is already part of `hasImageAttributes`.